### PR TITLE
Made requested changes

### PR DESCRIPTION
**Pull request by Imnaseli**

### CODE REVIEW

**Review by nzaih1999 — APPROVED** [review]
Hello @Imnaseli  thanks for your contribution. Will merge it real soon. Happy coding.

File: index.html [hunk 1]
```diff
@@ -114,7 +114,32 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+
         
+         <!-- ________ Imnaseli card START ________  -->
+                <div class="card">
+                  <p class="name">Imnaseli</p>
+                  <p class="contact">
+                    <i class="fab fa-twitter"></i>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
                    <i class="fab fa-github"></i>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
kindly do the following change and commit!

**Comment by TenTraicion** [comment]
Hello @Imnaseli, your card has already added to the project via #1585. It can be found in [this link](https://syknapse.github.io/Contribute-To-This-Project/)!

**Pull request closed without merging**